### DISCUSSION TITLE
yarn install error

### DISCUSSION THREAD

**Discussion by harvindernanda** [medusajs/medusa#5141] [Q&A]
While running yarn install to install the dependencies I am facing this issue. I am on Windows 11 

Link step
➤ YN0001: │ Error: While cloning /C:/Documents/Medusa/medusa/packages/oas/openapi-typescript-codegen/node_modules/jest-environment-jsdom -> /C:/Documents/Medusa/medusa/packages/medusa-file-minio/node_modules/jest-environment-jsdom UNKNOWN: unknown error, link 'C:\Documents\Medusa\medusa\packages\oas\openapi-typescript-codegen\node_modules\jest-environment-jsdom\LICENSE' -> 'C:\Documents\Medusa\medusa\packages\medusa-file-minio\node_modules\jest-environment-jsdom\LICENSE'
➤ YN0000: └ Completed in 2m 15s
➤ YN0000: Failed with errors in 3m 44s

Can you plse advise how to fix this?

**Comment by pevey** [comment]
Medusa is not developed or tested on Windows.  It may work, but some things may break unexpectedly.  Try it on WSL.
